src/VisualStudio/Core/Def/ValueTracking/ValueTrackingCommandHandler.cs

```diff
@@ -58,7 +59,8 @@ internal class ValueTrackingCommandHandler : ICommandHandler<ValueTrackingEditor
             IEditorFormatMapService formatMapService,
             IGlobalOptionService globalOptions,
             IAsynchronousOperationListenerProvider listenerProvider,
-            IUIThreadOperationExecutor threadOperationExecutor)
+            IUIThreadOperationExecutor threadOperationExecutor,
+            VisualStudioWorkspace workspace)
```

Comment:
Am I able to invoke value tracking from interactive windows or anything else?

Reply:
should not be able to